pkgs/top-level/python-packages.nix

```
@@ -3802,6 +3802,8 @@ in {
 
   retry_decorator = callPackage ../development/python-modules/retry_decorator { };
 
+  Quamash = callPackage ../development/python-modules/quamash { };
```

```suggestion
  quamash = callPackage ../development/python-modules/quamash { };
```

We normalize attributes to https://www.python.org/dev/peps/pep-0503/#normalized-names